helm/network-policies-app/templates/cilium/allow-proxy.yaml

```diff
@@ -0,0 +1,27 @@
+{{- if .Values.allowEgressToProxy.enabled }}
+{{ if not .Values.allowEgressToDNS.enabled }}
+{{ fail "allowEgressToCoreDNS.enabled=true is required for allowEgressToProxy.enabled since DNS requests are captured by Cilium for the L7 toFQDNs feature. Make sure the same namespaces are allowed for DNS egress." }}
```

Comment:
```suggestion
{{ fail "allowEgressToDNS.enabled=true is required for allowEgressToProxy.enabled since DNS requests are captured by Cilium for the L7 toFQDNs feature. Make sure the same namespaces are allowed for DNS egress." }}
```